### ISSUE TITLE
tinycc + Twitter issues

### ISSUE THREAD

**Issue by jthomasmock** [ottlngr/tinycc#1]
Just a heads up - tiny.cc links appear to make Twitter angry.

Not an issue with your package (it works great!) but wanted you to be aware in case anyone opens up another issue later on.

If tweeting from within R via `rtweet` you get a `This request looks like it might be automated. To protect our users from spam and other malicious activity, we can't complete this action right now. Please try again later.` Error message.

If tweeting from Twitter app or web-app - you just get a generic failure message.

More information on the [Twitter Dev Forums](https://twittercommunity.com/t/request-is-looks-like-automated/114586).

**Comment by ottlngr** [comment]
Thanks for the hint - read on Twitter earlier that you seem to have troubles with `tinycc` while automating tweets. I will add an appropriate note to the README or the like.

**Comment by jthomasmock** [comment]
Yup, it's definitely on Twitter's end, the package works like an absolute charm otherwise!
